frontend/src/metabase/admin/settings/notifications/WebhookForm.tsx

```diff
@@ -0,0 +1,212 @@
+import type { FormikHelpers } from "formik";
+import { t } from "ttag";
+import * as Yup from "yup";
+
+import { useTestChannelMutation } from "metabase/api/channel";
+import {
+  Form,
+  FormChipGroup,
+  FormProvider,
+  FormSubmitButton,
+  FormTextInput,
+} from "metabase/forms";
+import { FormKeyValueMapping } from "metabase/forms/components/FormKeyValueMapping";
+import { useActionButtonLabel } from "metabase/hooks/use-action-button-label";
+import { Button, Chip, Flex, Alert, Text, Group, Icon } from "metabase/ui";
+import type { NotificationAuthMethods } from "metabase-types/api";
+
+const validationSchema = Yup.object({
+  url: Yup.string()
+    .url(t`Please enter a correctly formatted URL`)
+    .required(),
```

Comment:
I think this also needs a localized string

```suggestion
    .required(t`Please enter a correctly formatted URL`),
```

Otherwise the yup library provides an English string, even in other locales

![image](https://github.com/user-attachments/assets/9f0767ba-95b2-424b-8089-e1c2441ce01d)

![image](https://github.com/user-attachments/assets/92b8adb7-cafa-47a1-a55c-43527f3f2e01)